docs/technical_reference/property_models/basic_water_properties.rst

```diff
@@ -0,0 +1,95 @@
+.. _basic_water_prop_ref:
+
+Basic Water Property Package
+============================
+
+The basic water property package is meant to calculate the most basic of water properties for WaterTAP unit models
+from the volumetric flow rate and the component mass concentration.
+All components are assumed to be in the liquid phase and properties are indexed only to component where appropriate.
```

Comment:
My only thought is what differentiates this from the basic water property model in WaterTAP main? Is it only the usage of concentration and volumetric flow as state vars in place of component mass flows? Or are there other differences? No need to change anything now, but want to know whether we should create an issue to eventually consolidate the two.

Reply:
Pretty much nothing. I anticipated getting more models from NMSU that would be able to make use of this prop pack, but that has not been the case (I thought they would be simpler). And using flow and concentration as state variables is more intuitive to me (as a non-ChemE). So I thought I was doing myself a favor.

The only model that uses this prop pack is chem softening, and in hindsight I should have just used one of the existing property packages. Now, using chemical softening in a flowsheet with any other property package requires translator blocks, so ended up making more work for us. What we should really do (not now) is convert that model to use a different property package.

Reply:
Oh by "basic property model" do you mean the one used for ZO models? Answer is the same, just curious. 

It might be easier to just use that ZO property model for chem softening model. 

(Sidebar: we have two property packages that are imported as ``WaterParameterBlock`` in WaterTAP main - that might be something worth making an issue about)